corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -677,6 +678,28 @@ cloudCare.AppView = Backbone.View.extend({
         data.onload = function (adapter, resp) {
             cloudCare.dispatch.trigger("form:ready", form, caseModel);
         };
+        data.answerCallback = function(sessionId) {
+
+            var rawBaseUrl = self.options.urlRoot;
```

Comment:
This seems very fragile. you can use the existing style by declaring the urls like this:
``` 
var renderXmlRoot = "{% url "cloudcare_render_xml" domain '' %}";
var renderFormRoot = "{% url "cloudcare_render_form" domain '' %}";
```
here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cloudcare/templates/cloudcare/cloudcare_home.html#L63
and then passing them into the app